deploy/05_deploy_asset_pool.js

```diff
@@ -0,0 +1,40 @@
+module.exports = async ({ getNamedAccounts, deployments }) => {
+  const { deploy, getArtifact, log, save } = deployments
+  const { deployer, rewardManager } = await getNamedAccounts()
+  const KeepToken = await deployments.get("KeepToken")
+  const UnderwriterToken = await deployments.get("UnderwriterToken")
+  const RewardsPool = await getArtifact("RewardsPool")
+
+  const AssetPool = await deploy("AssetPool", {
+    from: deployer,
+    args: [KeepToken.address, UnderwriterToken.address, rewardManager],
+    log: true,
+  })
+
+  const assetPool = await ethers.getContractAt("AssetPool", AssetPool.address)
+  const rewardsPoolAddress = await assetPool.rewardsPool()
+
+  // The`RewardsPool` contract is created in the `AssetPool` constructor so
+  // it's created in the same transaction.
+  const receipt = AssetPool.receipt
+  receipt.contractAddress = rewardsPoolAddress
```

Comment:
Is there a reason we modify the receipt? IMHO we should leave `contractAddress` unchanged as this was the original transaction.

Reply:
I updated it in e1020703a1f8dbfacd499c1d88d9ef546623a680. Please let me know if this shouldn't be changed.